Assets/MRTK/Providers/Oculus/XRSDK/MRTK.ThirdParty/MRTK-Quest/Editor/MRTKQuestConfigurationChecker.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace prvncher.MRTKQuest.Editor
```

Comment:
Since we are bringing this into MRTK, we should consider standardizing on the namespace.